src/components/header/topbar.style.scss

```diff
@@ -24,10 +24,23 @@
       margin-right: 10px;
     }
   }
+
+  .logo-href {
+    padding: 8px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &:hover {
+      background-color: #f9e8d3;
+      border-radius: 5px;
```

Comment:
border radius pe hover? why?

Reply:
Pentru ca unii dintre noi suntem la început și încă învățăm. Deci era mai bine lăsat în clasa border-radius-ul. Got it :) thanks

Reply:
poi, te rog modifica :)